src/tie_breaks/individual.py

```diff
@@ -389,7 +397,8 @@ def sum_of_buchholz(
     opponents: list[Player] = [
         tournament.players_by_id.get(pairing.opponent_id)
         for round_index, pairing in player.pairings.items()
-        if round_index < max_round
+        if round_index <= max_round
+        and pairing.opponent_id is not None
```

Comment:
The filtering is done line 412 already, why add it here?
I specifically used `get` as well (maybe the typing should reflect that)